.gitignore

```diff
@@ -9,6 +9,7 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+/.idea
```

Comment:
Cloud you remove `/.idea`  because IDE is different for each person. Please set it in your own `gitignore_global`.

Reply:
Reverted .gitignore, and also added a `hickory-dns` feature for `bigquery` crate.